cmd/utils/flags.go

```diff
@@ -1639,11 +1639,11 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 		}
 		cfg.Genesis = core.DefaultGoerliGenesisBlock()
 		SetDNSDiscoveryDefaults(cfg, params.GoerliGenesisHash)
-	case ctx.GlobalBool(YoloV2Flag.Name):
+	case ctx.GlobalBool(YoloV3Flag.Name):
 		if !ctx.GlobalIsSet(NetworkIdFlag.Name) {
-			cfg.NetworkId = 133519467574834 // "yolov2"
+			cfg.NetworkId = 133519467574834 // "yolov3"
```

Comment:
I also made this mistake when updating `yolov1` to `yolov2`. But actually: 
```
>>> import codecs
>>> codecs.decode(hex(133519467574834)[2:], "hex")
b'yolov2'
```
So this should be changed from `133519467574834` to `133519467574835`

Reply:
Or, even better, you can fix it so we don't do the same mistake again, and change it t
```
new(big.Int).SetBytes([]byte("yolov3")).Uint64()
```